_data/images/carte_noire_features.yml

```diff
@@ -0,0 +1,4 @@
+bunny:
+  src: https://i.imgur.com/DMCHDqF.jpg
+  alt: "Thumper"
+  external: true
```

Comment:
You need to put an empty line to the bottom of the file... this is also true for the following files.